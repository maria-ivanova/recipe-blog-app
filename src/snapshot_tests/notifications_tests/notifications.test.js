import React from 'react';
import renderer from 'react-test-renderer';
import Routing from '../helpers/router.js';
import Notifications from '../../components/notifications.js';

describe('Notifications Component', () => {
    it('should render layout', () => {
        const tree = renderer.create(
            <Routing>
                <Notifications />
            </Routing>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })
})