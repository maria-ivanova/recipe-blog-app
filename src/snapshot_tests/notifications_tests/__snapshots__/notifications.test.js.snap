// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notifications Component should render layout 1`] = `
<div
  className="notifications 
                             null 
                             "
>
  <span>
    
  </span>
</div>
`;
